docs/InstrumentationVerification/InstrumentationVerification.md

```diff
@@ -26,11 +26,19 @@ To run a verification, simply:
 
 
 
-1. Add the environment variable `DD_TRACE_WRITE_INSTRUMENTATION_TO_DISK`, set to “true”.
-2. Run or restart the application we are instrumenting. Observe that as we instrument the app, more and more files will appear under [Datadog Logs Folder]/InstrumentationVerification/{ProcessName}_{ProcessID}_{ProcessCreationTime}
-3. Via command line, run the command: 
-
-    Datadog.InstrumentedAssemblyVerification.Standalone.exe [Full path of the folder generated in step 2] 
+1. Verify that the environment variable `DD_TRACE_WRITE_INSTRUMENTATION_TO_DISK` is set to “true” (it should be on by default).
+2. Run your application. Observe that as we instrument the app, more and more files will appear under [Datadog Logs Folder]/InstrumentationVerification/{ProcessName}_{ProcessID}_{ProcessCreationTime}
+3. In order to run the analysis:
+    1. To analyze a running process:
+        Via command line, run the command: `dd-trace analyze-instrumentation [process name] [pid]`
+    2. To analyze a process that is no longer running (crashed/terminated):
+        Via command line, run the command: `dd-trace analyze-instrumentation [process name]`
+
+        **Note:** In this case the analysis will perform on the most recent log directory. To analyze a specific process, provide also the process ID.
```

Comment:
```suggestion
```

This kind of goes without saying I think?

Reply:
Fixed in 59dc3c9f22d7700ec54e60483dd11a150535c39c